app/ProjectAanvraag/Listeners/SyncWidget.php

```diff
@@ -38,7 +38,7 @@ public function __construct(
         private readonly ContactRepository $contactRepository,
         private readonly UiTiDv1ConsumerRepository $uiTiDv1ConsumerRepository,
         private readonly int $groupId,
-        private readonly Auth0UserRepository $auth0UserRepository,
+        private readonly UserRepository $auth0UserRepository,
```

Comment:
I would also rename the variable `$auth0UserRepository` to a more generic term for consistency.